pkg/apis/pipeline/v1alpha1/verificationpolicy_validation.go

```diff
@@ -61,13 +61,24 @@ func (vs *VerificationPolicySpec) Validate(ctx context.Context) (errs *apis.Fiel
 // Validate KeyRef will check if one of KeyRef's Data or SecretRef exists, and the
 // Supported HashAlgorithm is in supportedSignatureAlgorithms.
 func (key *KeyRef) Validate(ctx context.Context) (errs *apis.FieldError) {
-	if key.Data == "" && key.SecretRef == nil {
-		errs = errs.Also(apis.ErrMissingOneOf("data", "secretref"))
+	keyCount := 0
```

Comment:
Please add a comment explaining that you are validating that one and only one of Data, SecretRef, KMS is defined. (As the code exists today, I have to read and understand the code to figure that out myself -- please make my life easier!)

Reply:
sure!